### ISSUE TITLE
question: IAMPolicy only has arn property

### ISSUE THREAD

**Issue by mrkenkeller** [rebuy-de/aws-nuke#572]
Unlike most resources IAMPolicy does not have a name property, just an ARN property?  Is that for a reason that I'm missing?  

```golang
type IAMPolicy struct {
	svc *iam.IAM
	arn string
}
```

Example output:

```shell
global - IAMPolicy - arn:aws:iam::XXXXXXXXXXX:policy/my-test-policy - would remove
```
Since it did not match anything else I saw, I thought I'd check in with the creators before I thought too much about changing it.

**Comment by mrkenkeller** [comment]
I guess I should note I'm asking because I'd happily go change it to work with a name property, if that is not a problem.

**Comment by svenwltr** [comment]
The properties are missing, because the IAMPolicy resource was introduced before there was a property feature in *aws-nuke*. And apparently no one added them until now.

Feel free to open a PR with additional properties.

**Comment by der-eismann** [comment]
Closed since #593 was merged.
